_posts/2024-12-04-enable-pluggable-storage-in-opensearch-vectordb

```diff
@@ -0,0 +1,211 @@
+---
+layout: post
+title:  "Deep dive: Enable Pluggable Storage in OpenSearch VectorDB"
+authors:
+ - kdooyong
```

Comment:
you might want to add yourself in authors folder.

Reply:
oh, thank you. will update in the next rev